frontend/sw360-portlet/src/main/resources/META-INF/resources/html/moderation/view.jsp

```diff
@@ -363,71 +380,60 @@ AUI().use('liferay-portlet-url', function () {
+                    {title: "<liferay-ui:message key="type" />", data: "componentType", className: 'text-nowrap'},
+                    {title: "<liferay-ui:message key="document.name" />", render: {display: detailUrl}, data: "documentName"},
+                    {title: "<liferay-ui:message key="requesting.user" />", data: "requestingUser"},
+                    {title: "<liferay-ui:message key="department" />", data: "requestingUserDepartment"},
+                    {title: "<liferay-ui:message key="moderators" />","orderable": false , width: "10%", data: "moderators", render: {display: renderModeratorsListExpandable}},
+                    {title: "<liferay-ui:message key="state" />", data: "moderationState", className: 'text-nowrap'},
+                    {title: "<liferay-ui:message key="actions" />", data:"isClearingAdmin", "defaultContent": "", render: {display: closedModAction}, className: 'one action'}
                 ],
                 language: {
                     url: "<liferay-ui:message key="datatables.lang" />",
-                    //emptyTable: "<liferay-ui:message key="no.moderation.requests.found" />",
                     loadingRecords: "<liferay-ui:message key="loading" />"
                 },
-                initComplete: datatables.showPageContainer
+                fnDrawCallback: function(settings){
+                    closedModerationRequestsSize = settings.json.closedModerationRequests;
+                    moderationRequestsSize = settings.json.moderationRequests;
+                    datatables.showPageContainer;
+                    $(tableId + ' .TogglerModeratorsList').on('click', toggleModeratorsList );
+                    docReady();
+                },
+                "order": [[ 0, "desc" ]],
             }, [0,1,2,3,4,5,6], [7]);
         }
 
+        function closedModAction(isClearingAdmin, type, row) {
+            let deleteIcon = '<div class="actions"><svg class="delete lexicon-icon" data-moderation-request="' + row.id + '" data-document-name="' + row.documentName +'"><title><liferay-ui:message key="delete" /></title><use href="/o/org.eclipse.sw360.liferay-theme/images/clay/icons.svg#trash"/></svg></div>';
+            let successIcon = '<span class="badge badge-success">READY</span>';
+            
+            if(isClearingAdmin) {
+                return deleteIcon;
+            }
+            else if(isClearingAdmin == false) {
+                return successIcon;
+            }
+            return "";
+        }
+
+        function detailUrl(name, type, row) {
+            let url = makeModerationViewUrl(row.id);
+            let viewUrl = $("<a></a>").attr("href",url).css("word-break","break-word").text(name);
+            return viewUrl[0].outerHTML;
+        }
+
+        var homeUrl = themeDisplay.getURLHome().replace(/\/web\//, '/group/');
```

Comment:
please refer `link.js` for url formation

Reply:
done